docs/configuration/exclude-users-tasks.md

```diff
@@ -7,7 +7,6 @@ description: This section describes how to exclude certain users from being able
 The Governance-Excluded Users settings in Syskit Point allow you to exclude certain users receiving any of the following Governance tasks or task-related e-mails:
 
 * Access Review tasks 
-* Lifecycle Management tasks
```

Comment:
zamijeniti Access Review sa Workspace review
Lifecycle Management zamijeniti sa Inactive Workspaces
to su sad sve policyi, al svejedno može pisati bez obzira na to